test/fixtures/render/nullable.json

```diff
@@ -0,0 +1,410 @@
+{
+  "_version": "4.0",
+  "metadata": [],
+  "name": "",
+  "description": "",
+  "element": "category",
+  "resourceGroups": [
+    {
+      "name": "",
+      "description": "",
+      "resources": [
+        {
+          "element": "resource",
+          "name": "",
+          "description": "",
+          "uriTemplate": "/",
+          "model": {},
+          "parameters": [],
+          "actions": [
+            {
+              "name": "",
+              "description": "",
+              "method": "GET",
+              "parameters": [],
+              "attributes": {
+                "relation": "",
+                "uriTemplate": ""
+              },
+              "content": [],
+              "examples": [
+                {
+                  "name": "",
+                  "description": "",
+                  "requests": [],
+                  "responses": [
+                    {
+                      "name": "200",
+                      "description": "",
+                      "headers": [
+                        {
+                          "name": "Content-Type",
+                          "value": "application/json"
+                        }
+                      ],
+                      "body": "{\n  \"s1\": \"abc\",\n  \"s2\": \"def\",\n  \"s3\": \"\",\n  \"s4\": null,\n  \"s5\": null,\n  \"s6\": null,\n  \"s7\": \"xyz\",\n  \"s8\": \"xyz\"\n}",
```

Comment:
If that is incorrect should their be an issue created, since that behaviour is not part of my changes. I could probably look into fixing if it is actually a bug.